IdnoPlugins/StaticPages/templates/default/staticpages/admin.tpl.php

```diff
@@ -34,7 +34,7 @@
 <div class="row">
     <div class="col-md-10 col-md-offset-1">
         <p class="pages">
-            <a href="<?= \Idno\Core\Idno::site()->config()->getDisplayURL() ?>staticpages/edit/?category=<?= urlencode($category) ?>"
+            <a href="<?= \Idno\Core\Idno::site()->config()->getDisplayURL() ?>staticpages/edit/"
```

Comment:
No, but it was causing a notice error (and therefore an exception, since I run with pedantic mode on). Needed to tidy it up first, and didn't see much point in making this a second PR